src/window.c

```diff
@@ -2740,6 +2746,11 @@ win_close(win_T *win, int free_buf)
     else
 	clear_snapshot(curtab, SNAP_HELP_IDX);
 
+    if (bt_quickfix(win->w_buffer))
+	quickfix_window = true;
```

Comment:
TRUE/FALSE for int type.  true/false for bool type.

```suggestion
	quickfix_window = TRUE;
```